contents/blog/cheaper-analytics.md

```diff
@@ -0,0 +1,73 @@
+---
+date: 2024-03-21
+title: 'PostHog is now up to 80% cheaper'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: PostHog news
+tags:
+  - Product updates
+---
+
+## tl;dr
+
+* It is now possible to use PostHog's product and web analytics for up to 80% less, depending on your volumes. We can be used more cheaply than any popular competitive product.
+* We have a new event processing option that allows you to capture events without person information.
+* Previously we processed every event with person information.
+* This allows us to reduce our costs for events that do not need person information. We can pass this saving to our customers.
+* This means, if you use this option, we are by far the cheapest way for you to track your product compared to any major analytics competitor of ours
+
+## Pricing before and after
+
+| Tier (up to) | New event price without person information | Previous pricing / our new pricing for events with person information |
+| ---- | ----- | ----- | ----- |
+| 1M | $0 | $0 | $0 |
+| 2M | $0.0000198 |  $0.000248 |
+| 15M | $0.0000343 |  $0.000104 |
+| 50M | $0.0000295 |  $0.0000655 |
+| 100M | $0.0000218 |  $0.0000364 |
+| 250M | $0.0000150 | $0.0000187 |
+| ∞ | $0.000009 | $0.00001 |
+
+(we can offer further discounts if you are beyond 250M events/month regardless - at that scale, we suggest you talk to us)
```

Comment:
Is this line necessary? Makes me wonder why we'd even list a price at this level if we just go ahead and discount but not be transparent about the discount.

Maybe replace with:

```suggestion
(we can offer further discounts for those paying more than $2k/mo on an annual contract - at that scale, we suggest you talk to us anyway)
```